config.yaml

```diff
@@ -0,0 +1,16 @@
+access.cghub:
+access.gdc:
+access.ega: 
+access.icgc: 
+access.cghub:
+manifest.gdc:
+manifest.icgc:
+tool.icgc: /icgc/icgc-storage-client/bin/icgc-storage-client
+tool.ega: /icgc/ega-download-demo/EgaDemoClient.jar
+tool.gdc: /icgc/gdc-data-transfer-tool/gdc-client
+tool.cghub: /icgc/genetorrent/bin/gtdownload
+download.directory: /icgc/mnt/downloads
+udt: true
```

Comment:
We should talk about how to best name / organize these settings in the next iteration. We probably want to organize by tool if they are tool specific settings.